server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -897,12 +897,47 @@ public Owner updateOwner(@PathParam("owner_key") @Verify(Owner.class) String key
             }
         }
 
-        // Update the autobindDisabled field if the incoming value is null.
+        // Update the autobindDisabled field if the incoming value is not null.
         if (owner.getAutobindDisabled() != null) {
             toUpdate.setAutobindDisabled(owner.getAutobindDisabled());
         }
 
+        // Update the contentAccess field if the incoming value is not null.
+        if (owner.getContentAccessMode() != null) {
```

Comment:
What permissions are needed in order to update this? If the content access list can be updated via the REST API that should only be possible in hosted mode.  An owner should not be able to update this value for themselves.